src/elements/content-sidebar/activity-feed/common/activity-message/ShowOriginalButton.tsx

```diff
@@ -1,20 +1,13 @@
-/**
- * @flow
- * @file Show Original button component used by Comment Text component
- */
-
 import * as React from 'react';
 import { FormattedMessage } from 'react-intl';
-
 import PlainButton from '../../../../../components/plain-button';
-
 import messages from './messages';
 
-type Props = {
-    handleShowOriginal: Function,
-};
+export interface ShowOriginalButtonProps {
+    handleShowOriginal: (event: React.SyntheticEvent<HTMLButtonElement>) => void;
```

Comment:
Why did you choose `SyntheticEvent` over `MouseEvent`? What are the differences?